src/app/static/src/app/store/plotSelections/getters.ts

```diff
@@ -0,0 +1,16 @@
+import {ControlSelection, FilterSelection, PlotName, PlotSelectionsState} from "./plotSelections";
+import {FilterOption} from "../../generated";
+
+export const getters = {
+    controlSelectionFromId: (state: PlotSelectionsState) => (plotName: PlotName, controlId: string): FilterOption | undefined => {
+        return state[plotName].controls.find((f: ControlSelection) => f.id === controlId)?.selection[0]
+    },
+    filterSelectionFromId: (state: PlotSelectionsState) => (plotName: PlotName, filterId: string): FilterOption[] => {
+        const filterSelection = state[plotName].filters.find((f: FilterSelection) => f.filterId === filterId);
```

Comment:
??? probably, we think
```suggestion
        const filterSelection = state[plotName].filters.find((f: FilterSelection) => f.stateFilterId === filterId);
```